conf/formTemplateSchema.json

```diff
@@ -758,7 +758,8 @@
                     "type": "string"
                   },
                   "value": {
-                    "type": "string"
+                    "type": "string",
+                    "pattern": "^\\S+$"
```

Comment:
We cannot reject whitespaces altogether like this. Problem is that `value` can also include arbitrary expressions and so we need to do the whitespace check on more granular level. See parsing of `value` for more [clarity](https://github.com/hmrc/gform/blob/main/app/uk/gov/hmrc/gform/core/parsers/ValueParser.scala#L604-L610).

I'd say this granular check should be implemented as another check in [Verifier.scala](https://github.com/hmrc/gform/blob/main/app/uk/gov/hmrc/gform/formtemplate/Verifier.scala)

Reply:
Thanks for this info! Implementation has been changed